src/Plaid/PlaidClient.cs

```diff
@@ -320,43 +355,22 @@ private async Task<TResponse> BuildResponse<TResponse>(string url, HttpResponseM
 				if (ShowRawJsonValues)
 				{
 					var json = await response.Content.ReadAsStringAsync();
-					var result = JsonConvert.DeserializeObject<TResponse>(
-						json,
-						new JsonSerializerSettings()
-						{
-							DateFormatString = "yyyy-MM-dd",
-							NullValueHandling = NullValueHandling.Ignore,
-							Converters =
-							{
-								new EnumMemberEnumConverter(),
-							},
-						});
+					var result = JsonSerializer.Deserialize<TResponse>(json, options: JsonSerializerOptions);
 					result!.RawJsonForDebugging = json;
 					result.StatusCode = response.StatusCode;
```

Comment:
result!.StatusCode ... missing the ' ! ' (to be consistent with line 367)

Reply:
static flow analysis means only the first dereference of a variable in any code path needs the `!`. Here, that is done on the line before, so on this line, compiler (and us) know that if it got here, result is for sure not-null and no need for the !

Reply:
huh. Interesting. The compiler is pretty smart =)